src/NServiceBus.Core/Transports/TransportInfrastructure.cs

```diff
@@ -30,6 +30,6 @@ public IMessageReceiver GetReceiver(string receiverId)
         /// <summary>
         /// Disposes all transport internal resources.
         /// </summary>
-        public abstract Task DisposeAsync();
+        public abstract Task Shutdown();
```

Comment:
Would it make sense to make this non abstract and return Task.Completed in the base for transports that do not care about this method?

Reply:
It might be worthwhile being very clear about the fact that transport doesn't have any resources to clean up when shutting down. Given this is not a directly user-facing API where we save users from writing boilerplate code, I'd stick with the current, mandatory implementation but I don't have a strong opinion on this. Happy to discuss this further if you see more value in this idea.